types/chart.js/index.d.ts

```diff
@@ -168,6 +169,7 @@ declare namespace Chart {
         x?: number | string | Date;
         y?: number;
         r?: number;
+        t?: number;
```

Comment:
Docs say this could be anything Moment.js accepts. Could you please include `string` and `Date`?

Reply:
Done.